src/core/server/saved_objects/mappings/index_mappings.test.ts

```diff
@@ -108,89 +110,89 @@ describe('server/mapping/index_mapping', function () {
             y: {
               properties: {
                 z: {
-                  type: 'text'
-                }
-              }
-            }
-          }
-        }
+                  type: 'text',
+                },
+              },
+            },
+          },
+        },
       });
     });
 
     it('throws if any of the new properties conflict', () => {
       const initialMapping = {
-        root: { properties: { foo: 'bar' } }
+        root: { properties: { foo: { type: 'string' } } },
       };
       const extensions = [
         {
           properties: {
-            foo: 'bar'
-          }
-        }
+            foo: { type: 'string' },
```

Comment:
Now that we're defining the EsMapping type, each property (under the root elasticsearch mapping type) has to be an object.